ci/release/update-version.sh

```diff
@@ -35,9 +35,9 @@ function sed_runner() {
 echo "${NEXT_FULL_TAG}" > VERSION
 
 # bump RAPIDS libs
-sed_runner "/- librmm =/ s/=.*/=${NEXT_RAPIDS_VERSION}/g" conda/recipes/ucxx/meta.yaml
-sed_runner "/- rmm =/ s/=.*/=${NEXT_RAPIDS_VERSION}/g" conda/recipes/ucxx/meta.yaml
-sed_runner "/- rapids-dask-dependency =/ s/=.*/=${NEXT_RAPIDS_VERSION}/g" conda/recipes/ucxx/meta.yaml
+sed_runner "/- librmm =/ s/=.*/=${NEXT_RAPIDS_VERSION},>=0.0.0a0/g" conda/recipes/ucxx/meta.yaml
+sed_runner "/- rmm =/ s/=.*/=${NEXT_RAPIDS_VERSION},>=0.0.0a0/g" conda/recipes/ucxx/meta.yaml
+sed_runner "/- rapids-dask-dependency =/ s/=.*/=${NEXT_RAPIDS_VERSION},>=0.0.0a0/g" conda/recipes/ucxx/meta.yaml
```

Comment:
We use a slightly different approach to handling the RAPIDS-like / UCX-like version schemes elsewhere. In raft-dask, for instance, we have [a `conda_build_config.yaml` variable named `ucxx_version`](https://github.com/rapidsai/raft/blob/8fbce933bff14deee582eab345fb768f35cf7f27/conda/recipes/raft-dask/conda_build_config.yaml#L19-L23) and [we update that in one place in `update-version.sh`](https://github.com/rapidsai/raft/blob/8fbce933bff14deee582eab345fb768f35cf7f27/ci/release/update-version.sh#L76-L77).

Could we do something similar for `ucxx`? Rather than replacing the version for each RAPIDS library, let's add a single variable like `rapids_version` in [`conda_build_config.yaml`](https://github.com/rapidsai/ucxx/blob/branch-0.39/conda/recipes/ucxx/conda_build_config.yaml) and use that everywhere in `meta.yaml`.

This could be done in a follow-up PR, but I'd like to see it fixed in this release. (Same for ucx-py, probably, though I haven't checked.)

Reply:
Love that idea, thank you! That change seemed small enough that I pushed it here. Will look at doing something similar on https://github.com/rapidsai/ucx-py/pull/1044.